pkg/runtime-tools/generate/generate_suite_test.go

```diff
@@ -322,6 +344,15 @@ func withMemorySwap(v int64) specOption {
 	}
 }
 
+func withOomScoreAdj(v int) specOption {
```

Comment:
nit... also make v an optional pointer to int here in this with test func and

spec.Process.OOMScoreAdj = Int(value) // using Int(value) from ./options.go to optionally allocate a pointer to normalized copy of value


and add a test case that uses nil down in the suite test  pls :)

Reply:
v has become a pointer. The test case to nil down ha been added :)

Reply:
thx!****